sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5418,12 +5421,28 @@ piEnqueueKernelLaunch(pi_queue Queue, pi_kernel Kernel, pi_uint32 WorkDim,
     // No lock is needed here, unlike the immediate commandlist case above,
     // because the kernels are not actually submitted yet. Kernels will be
     // submitted only when the comamndlist is closed. Then, a lock is held.
-    ZE_CALL(zeCommandListAppendLaunchKernel,
+    zeResult = ZE_CALL_NOCHECK(zeCommandListAppendLaunchKernel,
             (CommandList->first, Kernel->ZeKernel, &ZeThreadGroupDimensions,
              ZeEvent, (*Event)->WaitList.Length,
              (*Event)->WaitList.ZeEventList));
   }
 
+  if(zeResult != ZE_RESULT_SUCCESS){
+	uint32_t sourceAttributesLength = 0U;
```

Comment:
Please follow code conventions:
```suggestion
	uint32_t SourceAttributesLength = 0U;
```

Reply:
ok